packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -619,6 +619,17 @@ export default class BackupNg {
         const jobId = job.id
 
         const remoteIds = unboxIdsFromPattern(job.remotes)
+        for (const id of remoteIds) {
+          const remote = await app.getRemote(id)
+          if (remote.proxy !== job.proxy) {
+            throw new Error(
+              job.proxy !== undefined
+                ? `The remote ${remote.name} should be linked to the proxy ${job.proxy}`
```

Comment:
What if the job is not linked to a proxy?

Reply:
It will fallback to the second message.

Reply:
My bad :+1: